Core/src/main/java/com/plotsquared/core/PlotSquared.java

```diff
@@ -290,11 +290,11 @@ public void loadCaptionMap() throws Exception {
 
     public void startExpiryTasks() {
         if (Settings.Enabled_Components.PLOT_EXPIRY) {
-            ExpireManager.IMP = new ExpireManager(this.eventDispatcher);
-            ExpireManager.IMP.runAutomatedTask();
+            ExpireManager manager = new ExpireManager(this.eventDispatcher);
```

Comment:
Should this not be accessing the expiry manager from guice?

Reply:
@dordsor21 I didn't want to call the `ExpireManager` from guice since I don't believe it allows for `eventDispatcher` to be explicitly set, which could impact `runAutomatedTask()`. If this isn't an actual concern, then I'll can revert this change or call `PlotSquared.platform().expireManager()` here instead.